.circleci/config.yml

```diff
@@ -19,7 +19,7 @@ executors:
           POSTGRES_DB: ttasmarthub
   machine-executor:
     machine:
-      image: circleci/classic:latest
+      image: ubuntu-2004:current
```

Comment:
Do you know the difference between the `ubuntu-2204` and `ubuntu-2004` images? Reading through their docs provided me no insight ha... https://circleci.com/developer/images?imageType=machine

Reply:
Looks like you have to dive into the forums to get the details
https://discuss.circleci.com/t/ubuntu-linux-vm-machine-images-2022-april-q2-update/43749

Nothing major jumps out to me software wise, we could try deploying with 22 if you want

Reply:
Oh it is ubuntu 20.04 vs 22.04. I think 20.04 is fine, just didn't get what the difference was.

Reply:
My mind was reading it as YYMM or something like that, so 2004 April of 2020 and 2204 was April of 2022. Which after I looked up release dates that is ubuntu is versioned... 14.04 came out in April of 2014, 16.04 April of 2016 (and so on). TIL